rust/release/action.yml

```diff
@@ -5,6 +5,10 @@ inputs:
   binary_name:
     description: "The name of the binary that will be built"
     required: true
+  working_directory:
```

Comment:
Super nitpicky but this action should be called `build`. `release` is a bit vague and this action will likely be used for more than just full releases

Reply:
I think a good compromise here is to use `build-and-upload`, since that is what it does. `build` on its own implies the artifact is built and nothing else.